source/configuration_manual/quota/quota_configuration.rst

```diff
@@ -0,0 +1,506 @@
+.. _quota_configuration:
+
+===================
+Quota Configuration
+===================
+
```

Comment:
There should be some easy way to jump between these quota pages. I think the index page should at the top of it link to this quota configuration (like it does now) but also link to the other quota pages. And quota configuration and all the other quota pages should also link to the quota index page.

Reply:
Reviewing this again ... it turns out we had 2 separate configuration pages (quota/index.rst and quota/quota_configuration.rst), that had pretty much the same info.  So I merged those two pages into a single page.  I added a link back to the main quota plugin page from the top of this merged page.  The main quota plugin page links to the configuration page.  I'm able to find all quota links by looking at one of the two pages (plugin page, configuration page), so the links are sufficient in my opinion now.